android/source/src/main/kotlin/com/gu/source/components/pager/PagerProgressBar.kt

```diff
@@ -54,7 +54,7 @@ private val DefaultUnSelectedIndicatorColour = AppColour(
     dark = Source.Palette.Neutral100.copy(alpha = 0.3f),
 )
 
-private val DefaultButtonColours = ButtonColours(
+val DefaultButtonColours = ButtonColours(
```

Comment:
This and other changes below - make them `internal` instead of `public`.

When developing a library, we need to be very conscious about what we make public - it affects the API surface of the library accidentally making implementations available to consumers.